lib_xua/src/core/buffer/decouple/decouple.xc

```diff
@@ -796,11 +832,11 @@ void XUA_Buffer_Decouple(chanend c_mix_out
 
             /* Check for freq change or other update */
 
-            GET_SHARED_GLOBAL(tmp, g_freqChange_flag);
+            GET_SHARED_GLOBAL(tmp, g_streamChange_flag);
```

Comment:
I'd probably take the opportunity to use a better var name here

Reply:
How about g_StreamChangeCommand? It's mixed command and flag really but more of a command with 0 = No command

Reply:
Sorry, meant for "tmp"

Reply:
renamed cmd